torch/_export/graph_module.py

```diff
@@ -124,49 +129,14 @@ def str_to_op(str_op: str):
     root = torch.nn.Module()
     root.__dict__ = body
 
-    exir_meta = body["meta"][EXPORT_METADATA]
-    gm = make_export_graph_module(
-        root, graph, in_spec=exir_meta.in_spec, out_spec=exir_meta.out_spec, class_name=body["_graphmodule_cls_name"]
-    )
+    gm = torch.fx.GraphModule(root, graph, class_name="ExportGraphModule")
+    gm.__class__ = type(type(gm).__name__, (ExportGraphModuleMixin, type(gm)), {})
 
     gm.recompile()
     return gm
 
 
 class ExportGraphModuleMixin:
```

Comment:
Sorry why do we still need this? Since you are implementing the serialization, it shouldn't call reduce anymore right?

Reply:
Yup, I removed it.